src/http_hook.js

```diff
@@ -31,7 +32,7 @@ const init = function (client, config) {
             res.on('finish', () => {
                 const labels = {
                     method: req.method,
-                    route: req.url,
+                    route: config.NORMALIZE_ENDPOINT ? normalizeEndpoint(req.url) : req.url,
```

Comment:
Do we want to normalize the endpoint by default ? I can't imagine a case where we would want to store request parameters as labels.

Reply:
Me neither, but, I don't want to potentially break the project of everyone who is already using it